### DISCUSSION TITLE
Security Vulnerability information per zephyr release

### DISCUSSION THREAD

**Discussion by PragatiGarg-eaton** [zephyrproject-rtos/zephyr#71930] [General] [answered]
Hi, 

I have one query related to 'Security Vulnerability; section information in zephyr release pages:

- I could find that [[Zephyr Release Notes — Zephyr Project Documentation](https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdocs.zephyrproject.org%2Flatest%2Freleases%2Frelease-notes-3.2.html%23security-vulnerability-related&data=05%7C02%7Cpragatigarg%40eaton.com%7Ced135fe12643428e392408dc65086393%7Cd6525c95b906431ab926e9b51ba43cc4%7C0%7C0%7C638496333932610982%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=Ulhvl4tTWo7jzSX9dOc5uqMjkLfWbMYGiP39KUXgMV0%3D&reserved=0)](https://docs.zephyrproject.org/latest/security/vulnerabilities.html) is a common document page added for all the vulnerabilities found till latest release, in the zephyr version release notes. 
- Instead of having a common page, shouldn't we have a version specific page for each release? Would love to hear your thoughts on this.
- If we have it already, please share.

**Comment by kartben** [comment]
cc @ceolin

**Comment by ceolin** [comment, marked as answer]
I am not sure how having it split by release would be helpful. Vulnerabilities tend found/fixed during a release cycle are problems affecting older versions (the current release has them fixed). So e.g., you look vulnerabilities in release 3.6, none of them are affecting 3.6 but they may be affecting all other older releases.

What may be helpful is a way to look a release and check which vulnerabilities are affecting that release and which ones were fixed.

I don't know ... maybe I am missing something here, please let me know.
